kustomize/components/clairpgupgrade/base/clair-pg.statefulset.patch.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: apps/v1
-kind: Deployment
+kind: StatefulSet
```

Comment:
how will this work with existing deployments? On upgrade will it replace the deployment with a statefulset?

Reply:
Yes, it will replace the deployment with a stateful set. This is so that we do not have issues with overlapping deployments that would cause issues on the upgrade job.